src/language-css/parser-postcss.js

```diff
@@ -45,6 +53,17 @@ function parseValueNodes(nodes, options) {
       node.unit = "...";
     }
 
+    if (node.type === "func" && node.value === "selector") {
+      node.group.groups = [
+        parseSelector(
+          getHighestAncestor(valueNode).text.slice(
+            node.group.open.sourceIndex + 1,
+            node.group.close.sourceIndex
+          )
+        ),
+      ];
+    }
```

Comment:
Very bad workaround, but I think we can't fix it on our side, so it is better than nothing